### PR TITLE
Update webhooks section

### PR DESCRIPTION
**Pull request by m-murasovs**
Preparing the Integrations section, so it's ready for the Appmixer and other Integration docs.
Don't worry about the many file changes, it's mostly link and image updates

Preview: https://docs.apify.com/integrations?version=feature/webhooks-update

### CODE REVIEW

File: content/docs/integrations/slack.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Click the **Start** button and head to the Slack channel you selected to see your first Apify integration notifications.
+
+## Step 4: Start your run directly from Slack
+
+You can now run the same task directly from Slack by typing /apify run [task name] into the Slack message box. Remember you can retrieve the task’s name by clicking Tasks in the left sidebar on Apify Console.
+
+![Use Apify from Slack]({{@asset integrations/images/slack-apify-message.webp}})
+
+When an actor doesn’t require you to fill in any input fields, you can run it by simply typing `/apify run [actor name]`.
```
Comment (valekjo) [inline content/docs/integrations/slack.md:48]: Wow, actually didn't know about this! Just it's `/apify call ...`

File: content/docs/integrations/slack.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## [](#step-two) Step 2: Set up the Slack integration
+
+Now click the **Integrations** tab, find the Slack integration, and click the **Configure** button. You will be prompted to log in with your Slack account and select your workspace in the **Settings > Integrations** window.
+
+![Integrations tab]({{@asset integrations/images/integrations-tab.webp}})
+
+Then, head back to your actor to finish the setup. Select what type of events you would like to be notified of (e.g., when a run is created, when a run succeeds, when a run fails, etc.), your workspace, and the channel you want to receive the notifications in (you can set up an ad-hoc channel for this test). In the **Message** field, you can see how the notification will look, or you can craft a new custom one.
```
Comment (valekjo) [inline content/docs/integrations/slack.md:32]:
```suggestion
Then, head back to your task to finish the setup. Select what type of events you would like to be notified of (e.g., when a run is created, when a run succeeds, when a run fails, etc.), your workspace, and the channel you want to receive the notifications in (you can set up an ad-hoc channel for this test). In the **Message** field, you can see how the notification will look, or you can craft a new custom one.
```

File: content/docs/integrations/slack.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+[Slack](https://slack.com/) allows you to install various services in your workspace in order to automate and centralize tasks. Apify is one of these services, and it allows you to run your Apify actors, be notified about their run status, and get your results, all without opening your browser.
+
+## Get started
+
+To use the Apify integration for Slack, you will need:
+
+- An [Apify account](https://console.apify.com/).
+- A Slack account (and workspace).
+
+## [](#step-one) Step 1: Create a task in Apify Console
```
Comment (valekjo) [inline content/docs/integrations/slack.md:20]: I didn't get why the task is created first, but is it for the purpose of showing how to run it later directly from slack?

**Review by valekjo — CHANGES_REQUESTED** [review]

File: content/docs/integrations/webhooks/actions.md [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
 title: Actions
 description: Send notifications when specific events occur in your actor (task) run or build. Dynamically add data to the notification payload when sending the notification.
-menuWeight: 11.2
+menuWeight: 2
 paths:
     - webhooks/actions
+    - integrations/webhooks/actions
 ---
 
+<!-- If the URL points toward APify, we don't need to add the token because it gets added automatically -->
```
[the remark this answers is not in the document]
Comment (m-murasovs) [reply content/docs/integrations/webhooks/actions.md:10]: Yep 😄

File: content/docs/integrations/slack.md (hunk 3 above)
Comment (valekjo) [reply content/docs/integrations/slack.md:20]: Yep, I think that should be ok.

**Review by valekjo — APPROVED** [review]

**Pull request merged**